multipledispatch/tests/test_variadic.py

```diff
@@ -0,0 +1,32 @@
+from multipledispatch.variadic import isvariadic, Variadic
+
+
+class A(object): pass
+class B(A): pass
+class C(object): pass
+
+
+def test_is_variadic():
+	assert isvariadic(Variadic[A])
+	assert not isvariadic(A)
```

Comment:
Indentation is looking a little weird here.

Reply:
defaulted to tabs on accident